### PR TITLE
`CNDIT-1067`: Update the build.sh to build the Person Docker Service

### PR DESCRIPTION
**Pull request by alog-enquizit**
- Update the build.sh to include the Person Reporting service in the local docker build
- Add log statements to the Kafka topic creation
- Change the Reporting Service default port to 8081.
- Health Check:
>-http://localhost:8081/status

### CODE REVIEW

File: data-reporting-service/build.sh [hunk 1]
```diff
@@ -25,15 +25,21 @@ echo "Building Kafka Broker"
 docker-compose -f $BASE/docker-compose.yml up broker --build -d
 
 # Build and deploy ETL Data pipeline container
-echo "Building ETL Data Pipeline"
+echo "Updating the th Data Pipeline"
```
Comment (upasanapattnaik-eq) [inline data-reporting-service/build.sh:28]: Can we keep the comment as "Building the-"

File: data-reporting-service/build.sh [hunk 2]
```diff
@@ -25,15 +25,21 @@ echo "Building Kafka Broker"
...
 docker-compose -f $BASE/docker-compose.yml up data-reporting-service --build -d
 
+echo "Updating the th Person Reporting Service"
```
Comment (upasanapattnaik-eq) [inline data-reporting-service/build.sh:31]: Small nitpick: Extra word

File: data-reporting-service/src/main/resources/application.yaml [hunk 3]
```diff
@@ -14,4 +14,4 @@ logging:
     name: PersonService.log
     path: ${DA_LOG_PATH:logs}
 server:
-  port: '8080'
+  port: '8081'
```
Comment (upasanapattnaik-eq) [inline data-reporting-service/src/main/resources/application.yaml:17]: Are the ports for the other services determined?

File: data-reporting-service/build.sh (hunk 1 above)
Comment (alog-enquizit) [reply data-reporting-service/build.sh]: Good catch!

File: data-reporting-service/build.sh (hunk 2 above)
Comment (alog-enquizit) [reply data-reporting-service/build.sh]: Done!

File: data-reporting-service/src/main/resources/application.yaml (hunk 3 above)
Comment (alog-enquizit) [reply data-reporting-service/src/main/resources/application.yaml:17]: 8080 is usually used by other services as default port. So making this to 8081. 
We are free to fix whatever port as per foundations team.

Comment (upasanapattnaik-eq) [reply data-reporting-service/src/main/resources/application.yaml:17]: Got it. Thanks Anand!

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM. Thank you for changes and clarifications.

**Pull request merged**